Tools/nib2cib/NSButton.j

```diff
@@ -439,6 +440,7 @@ var NSButtonIsBorderedMask = 0x00800000,
 
         _isBordered = (buttonFlags & NSButtonIsBorderedMask) ? YES : NO;
         _bezelStyle = (buttonFlags2 & 0x7) | ((buttonFlags2 & 0x20) >> 2);
+        _imageScaling = (((buttonFlags2 >> 6) & 0x3) + 2) % 4;
```

Comment:
I'm not happy with it neither. I know for sure that the OS X impl is ((buttonFlags2 >> 6) & 0x3) but with this I get the values 0,1,2,3 instead of 2,3,0,1. The solution is certainly simpler than that but currently i'm out  of ideas.